src/lib/operators/join_hash.cpp

```diff
@@ -388,6 +391,13 @@ void probe(const RadixContainer<RightType>& radix_container,
       PosList pos_list_left_local;
       PosList pos_list_right_local;
 
+      // simple heuristic to estimate result size: half of the partition's row will match
+      // a more conservative pre-allocation would be the size of the left cluster
+      // TODO: info is in the radix_container ...
```

Comment:
?

Reply:
Yeah, I wanted to start a "changes requested" review for that line, but that's not possible. I am on it.

Reply:
Addressed by removing TODO. Maybe later address the sizing when we find the current reserve to be too large or too small.